src/GuitarProToMidi.Console/Native/Format.cs

```diff
@@ -284,7 +287,8 @@ private List<Note> RetrieveNotes(GuitarProToMidi.Track track, int[] tuning, Trac
                 case SimileMark.none:
                     break;
                 default:
-                    throw new ArgumentOutOfRangeException(nameof(m.simileMark), m.simileMark, "Unknown enum value.");
+                    logger.Warn("Warning: Skipping unhandled data {0} {1} {2}", nameof(m.simileMark), m.simileMark, "Unknown enum value");
```

Comment:
The CI is complaining because there is no `logger` with a lowercase `l`. It's defined with an uppercase `L` instead.

Reply:
Thank you so much, I kinda missed this 👍